cob_hardware_config/cob3-6/config/teleop.yaml

```diff
@@ -1,49 +1,62 @@
+#cob3-6
+##########
 # common params
 run_factor: 2.0
+joy_num_buttons: 12
+joy_num_axes: 6
+joy_num_modes: 4
+mode_switch_button: 8
+light_topic_name: /light
+
+# axes
+axis_vx: 1
+axis_vy: 0
+axis_vz: 5
+axis_roll: 4
+axis_pitch: 3
+axis_yaw: 2
 
 # buttons
-lower_neck_button: 6
-upper_neck_button: 4
-tray_button: 3
-arm_joint12_button: 0
-arm_joint34_button: 1
-arm_joint56_button: 2
-arm_joint7_button: 3
-base_safety_button: 4
 deadman_button: 5
+safety_button: 4
+init_button: 9
+
+#mode1: Base
 run_button: 7
 
-# axes
-axis_vx: 1
-axis_vy: 0
-axis_vth: 2
-up_down: 5 #tray--up/down; tilt--front/back, here we just name up_down
-left_right: 4 #pan--left/right
+#mode2: Trajectory controller
+torso_sss_default_target_button: 1
+head_sss_default_target_button: 3
+arm_sss_default_target_button: 2
+
+#mode3: Velocity group controller
+arm_joint1_button: 0
+arm_joint2_button: 1
+arm_joint3_button: 2
+arm_joint4_button: 3
```

Comment:
Is it only possible to command 4 of 7 joints with the new teleop?

Could we use a bitmap-like approach to increase the number of joints? such as [0|1]*2^0 + [0|1]*2^1 + [0|1]*2^2 + [0|1]*2^3. This would give us the chance to command 15 joints

However, if we can only command maximum four joints __and__ the bitmap-like approach is not really intuitive to use, I'd rather remove this mode (i.e. single-joint-motion) from the teleop...